graphite/graphite_tcp.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // TCPServer processes Graphite data received over TCP connections.
 type TCPServer struct {
-	server Server
+	server SeriesWriter
```

Comment:
OK, I think I follow you. Let me take another pass at this. However, calling something `GraphiteServer` within the graphite package does not follow Go style.